QUEST_LV_0100.tsv

```diff
@@ -4871,74 +4871,74 @@ QUEST_LV_0100_20150317_004870	Hand over Big Cockatrice tails to Tara Miles
-QUEST_LV_0100_20150317_004923	Now, one more step to go before the Jewel of Prominence gets completed. Talk to Grita.
-QUEST_LV_0100_20150317_004924	Combine materials with the Jewel of Prominence in the flame generator
-QUEST_LV_0100_20150317_004925	Grita told you that since the Jewel of Prominence is not complete yet, you should combine materials with it in the flame generator. Combine the materials at the flame generator.
-QUEST_LV_0100_20150317_004926	You successfully completed the Jewel of Prominence. Talk to Grita.
-QUEST_LV_0100_20150317_004927	Grita is waiting at the third floor of Mage Tower. Talk to Grita.
+QUEST_LV_0100_20150317_004922	$Obtain Flame Sources
+QUEST_LV_0100_20150317_004923	$Now, one more step to complete the Jewel of Prominence. Talk to Grita.
+QUEST_LV_0100_20150317_004924	$Combine materials with the Jewel of Prominence in the Flame Fusion Machine
+QUEST_LV_0100_20150317_004925	$Grita told you that since the Jewel of Prominence is not complete yet, you should combine materials with it in the Flame Fusion Machine. Combine the materials at the Flame Fusion Machine.
+QUEST_LV_0100_20150317_004926	$You successfully completed the Jewel of Prominence. Talk to Grita.
+QUEST_LV_0100_20150317_004927	$Grita is waiting at the third floor of Mage Tower. Go to Grita.
 QUEST_LV_0100_20150317_004928	{memo X}Defeat Krumadok
 QUEST_LV_0100_20150317_004929	{memo X}It seems that the mad magician Antares is loose. Defeat the Krumadok which Antares has summoned.
 QUEST_LV_0100_20150317_004930	{memo X}You defeated Krumadoks which Antares summoned. Talk to Grita.
 QUEST_LV_0100_20150317_004931	It was the mad magician, Antares who summoned the Krumadoks. Talk to Grita.
-QUEST_LV_0100_20150317_004932	Destroy the first valve of magic
-QUEST_LV_0100_20150317_004933	Grita told you to break the valve of magic to block Antares. Destroy the valve at the lab room.
-QUEST_LV_0100_20150317_004934	As you destroyed the valve, it exploded and the monsters disappeared, but it was really dangerous. Talk to Grita again.
-QUEST_LV_0100_20150317_004935	Destroy the valve of magic
-QUEST_LV_0100_20150317_004936	The power of the valve of magic was way more than you expected. You better discuss it with Grita.
-QUEST_LV_0100_20150317_004937	Look for the info on the valve of magic
-QUEST_LV_0100_20150317_004938	Grita told you that since Antares researched on the flow of the magic, the research information must be left somewhere. Research the books of Antares and find the information on the magic device.
-QUEST_LV_0100_20150317_004939	It seems that you found information on the flow of the magic. Talk to Grita.
+QUEST_LV_0100_20150317_004932	$Destroy the first Magic Control Valve
+QUEST_LV_0100_20150317_004933	$Grita told you to break the Magic Control Valve to stop Antares. Destroy the valve at the Laboratory.
+QUEST_LV_0100_20150317_004934	$As you destroyed the valve, it exploded and the monsters disappeared, it was really dangerous. Talk to Grita again.
```

Comment:
Did the monsters disappear or appear?
"As you destroyed the valve, it exploded and monsters appeared, which was really dangerous."